ui/store/actions.js

```diff
@@ -1116,6 +1126,37 @@ export function updateMetamaskState(newState) {
       type: actionConstants.UPDATE_METAMASK_STATE,
       value: newState,
     });
+
+    // Check that the transaction was not submitted successfully, and remove it from failed transactions if it was.
+    const transactionIdsToRemove = Object.values(
```

Comment:
Was this meant to be `Object.keys`? The values are all booleans, not ids.

Reply:
yes object.keys.